lavalink/PlayerManager.py

```diff
@@ -108,6 +113,38 @@ def add(self, requester: int, track: dict):
             self.current = track
             await self._lavalink.ws.send(op='play', guildId=self.guild_id, track=track.track)
             await self._lavalink.dispatch_event(TrackStartEvent(self, track))
+            
+    async def play_now(self, requester:int, track):

```

Comment:
Repetition of the code it's not the best solution. You should rather: use function `add_next` to add it to the start of the queue and execute the `play` function. 2 lines of code. This eliminates the amount of code that needs to be fixed or changed in the future.